goethe/src/main/java/com/palantir/goethe/Goethe.java

```diff
@@ -18,7 +18,7 @@
 
 import com.google.common.base.Preconditions;
 import com.google.common.base.Splitter;
-import com.squareup.javapoet.JavaFile;
+import com.palantir.javapoet.JavaFile;
```

Comment:
I usually prefer to use both fully qualified names in situations like this, so it's always obvious which codepath is which